src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1876,6 +1876,15 @@ void primitive_inst::prepare_primitive() {
             return;
         }
 
+        if (_node->is_type<read_value>()) {
```

Comment:
We have very similar code here:
https://github.com/openvinotoolkit/openvino/blob/4e5ac62f51f1c26fd000922cfb7d142e8b71bed5/src/plugins/intel_gpu/src/graph/network.cpp#L758-L765
Can it be used to additionally apply SKIP flag to the read_value instance itself?

Reply:
For optimized `read_value` we will also need to call `update_output_memory()`, which will not happen if we set the `SKIP` flag in advance